index.js

```diff
@@ -12,5 +12,49 @@ function splitNumber(number){
         const parsedInt = Number.parseFloat(int) + 1;
         inputNumber.value = parsedInt.toFixed(2);
       }
-      
-  }
\ No newline at end of file
+    }
+
+function submitForm(){
+  try{
+    const inputNumber = document.getElementById("input-number");
+    const time = inputNumber.value;
+
+    const inputType = document.getElementById("input-type");
+    const type = inputType.value;
+
+    const inputDate = document.getElementById("input-date");
+    const date = inputDate.value;
+
+
+    if(time == 0 || date == ""){
+      throw new Error("Invalid data");
+    }
+    
+    insertData([time, type, date]);
+  }catch(err){
+    alert(`${err.message}`);
+  }
+
+}
+
+function insertData(data){
+  console.log(data);
+
+  const tr = document.createElement("tr");
+  tr.className = "info-table-body-tr";
+   
+    data.forEach(d => {
+      const td = document.createElement("td");
+      td.className = "info-table-body-tr-td";
+      td.innerText = d;
+      tr.appendChild(td);
+    });
+
+  const table = document.querySelector("#table");
+  const [tbody] = table.getElementsByTagName("tbody");
```

Comment:
Aqui voce pode usar o querySelector pra manter o padrão, e tambem usar o unitario (ao inves de um selectAll ja que sua tabela só vai ter um tbody)

```suggestion
  const tbody = table.querySelector("tbody");
```